assets/src/infrastructure/ui/display/Banner.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Alert as ChakraAlert, AlertIcon, AlertTitle, AlertDescription, AlertProps, IconProps } from '@chakra-ui/core';
+
+interface Props extends AlertProps {
+	description: string;
+	iconProps?: IconProps;
+	title: string;
+}
+
+const Banner: React.FC<Props> = ({ children, description, iconProps, title, ...props }) => (
+	<ChakraAlert flexDirection='column' justifyContent='center' textAlign='center' height='200px' {...props}>
+		<AlertIcon size='40px' mr={0} {...iconProps} />
```

Comment:
only suggestion is maybe to use rem instead of px as it supposedly scales better

```suggestion
		<AlertIcon size='2.5rem' mr={0} {...iconProps} />
```

2.5rem should equal 40px at the standard font scaling

Reply:
good one, thx :bow: